docs/storybook/pages/wall.stories.tsx

```
@@ -0,0 +1,389 @@
+                <AccordionBody>
+                  Chez Bouygues Telecom, nous vous accompagnons depuis plus de 25 ans dans le choix de votre téléphone
+                  mobile. Notre but : tout faire pour que vous trouviez le smartphone le plus adapté à votre quotidien.
+                  Besoin d'un téléphone pas cher pour aller avec votre forfait mobile ? Envie de craquer pour le
+                  meilleur smartphone 5G du moment ou simplement de profiter d'un bon plan smartphone ? A la recherche
+                  d'un smartphone reconditionné ? iPhone, Samsung, Xiaomi, OPPO, Huawei… Nous espérons que vous
+                  trouverez votre bonheur parmi les dizaines de smartphones que nous avons sélectionnés au sein des plus
+                  grandes marques.
+                </AccordionBody>
+              </AccordionItem>
+              <AccordionItem children>
+                <AccordionHeader>Quel smartphone choisir ?</AccordionHeader>
```

Biome (1.9.4)

[error] Avoid passing children using a prop

The canonical way to pass children in React is to use JSX elements

(lint/correctness/noChildrenProp)